addons/mass_mailing_themes/views/mass_mailing_themes_templates.xml

```diff
@@ -866,4 +881,750 @@
+                            </li>
+                            <li>
+                                <a href="#" target="_blank" data-original-title="" title="">Why You Need Two Types of Content Strategists</a>
+                            </li>
+                            <li>
+                                <a href="#" target="_blank" data-original-title="" title="">7 Ways to Nurture Creativity</a>
+                            </li>
+                            <li>
+                                <a href="#" target="_blank" data-original-title="" title="">Infographic: Our Company Throughout The Generations</a>
+                            </li>
+                            <li>
+                                <a href="#" target="_blank" data-original-title="" title="">7 Unexpected Signs You're Good At Your Job</a>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="s_hr o_mail_snippet_general pt16 pb0" style="background-color: rgb(238, 233, 226) !important;" data-snippet="s_hr" data-name="Separator">
+            <hr class="s_hr_1px s_hr_solid"/>
+        </div>
+        <div class="s_footer_social o_mail_block_footer_social o_mail_footer_social_center o_mail_snippet_general o_cc o_cc5 pt0 pb0" style="background-color: rgb(84, 52, 39) !important;" data-snippet="s_mail_block_footer_social" data-name="Footer Center">
+            <div class="container o_mail_table_styles">
+                <div class="row">
+                    <div class="col-lg-12 o_mail_footer_social">
+                        <a aria-label="Facebook" title="Facebook" target="_blank" href="https://www.facebook.com/Odoo"><span class="fa fa-facebook text-o-color-4"></span></a> &amp;nbsp;&amp;nbsp;
+                        <a style="margin-left:10px" aria-label="Twitter" target="_blank" title="Twitter" href="https://twitter.com/Odoo"><span class="fa fa-twitter text-o-color-4"></span></a>  &amp;nbsp;&amp;nbsp;
+                        <a style="margin-left:10px" aria-label="Instagram" title="Instagram" target="_blank" href="https://www.instagram.com/explore/tags/odoo/"><span class="fa fa-instagram text-o-color-4"></span></a>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-lg-12 o_mail_footer_links">
+                        <a role="button" href="/unsubscribe_from_list" class="btn btn-link text-o-color-4">Unsubscribe</a> | <a role="button" href="/contactus" class="btn btn-link text-o-color-4">Contact</a>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-lg-12">
+                        <p class="o_mail_footer_copy text-o-color-4">
+                            © <t t-esc="datetime.datetime.now().year"/> All Rights Reserved
```

Comment:
I'm curious because there is a t-set now at the top of every template but here we don't use it ?

Reply:
I tried to remove them but it created errors :/

Reply:
And you couldn't use it here ?